client/components/theme-upgrade-modal/index.tsx

```diff
@@ -73,7 +73,7 @@ export const ThemeUpgradeModal = ( {
 	const showBundleVersion = theme_software_set;
 	const isExternallyManaged = useSelector( ( state ) => isExternallyManagedTheme( state, slug ) );
 
-	const bundleSettings = useBundleSettings( slug );
+	const bundleSettings = useBundleSettingsByTheme( slug );
```

Comment:
I applied your PR in https://github.com/Automattic/wp-calypso/pull/85013, and I changed to use `useBundleSettings`, as I wanted to keep `ThemeUpgradeModal` Calypso-State-free. Could you apply it here as well? Thank you!

- https://github.com/Automattic/wp-calypso/pull/85013#discussion_r1424374607
- https://github.com/Automattic/wp-calypso/pull/85013/files#diff-37aa85681c6036ea769d4a6169a3d623a36082cdba5dcbcfc48bca50214fb4b6
```suggestion
	const bundleSettings = useBundleSettings( slug );
```

Reply:
Hey @okmttdhr! Of course!

Here is the change: https://github.com/Automattic/wp-calypso/pull/84886/commits/373f83edaaebf8beba4b852945d2980d4a68647a

Reply:
LGTM :) Thank you!